File: Exemplos SmartPOS/App_IntentDigitalHUB_Ionic/Elgin_IntentDigitalHUB_SmartPOS_Ionic-font/src/intentDigitalHubService/TERMICA/Commands/ImprimeXMLSAT.ts
import {TermicaCommand} from '../TermicaCommand';

export class ImprimeXMLSAT extends TermicaCommand {
  readonly dados: string;
  readonly param: number;

  constructor(dados: string, param: number) {
    super('ImprimeXMLSAT');
    this.dados = dados;
    this.param = param;
  }

  functionParameters(): object {
    return {dados: this.dados, param: this.param};
  }
}

File: Exemplos SmartPOS/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_SmartPOS_Flutter-font/lib/IntentDigitalHubService/TERMICA/Commands/imprime_xml_sat.dart
import '../termica_command.dart';

class ImprimeXMLSAT extends TermicaCommand {
  final String _dados;
  final int _param;

  ImprimeXMLSAT(this._dados, this._param) : super('ImprimeXMLSAT');

  @override
  get functionParametersJson => {
        'dados': _dados,
        'param': _param,
      };
}
